misc/python/materialize/bintray.py

```diff
@@ -103,12 +103,15 @@ def publish_uploads(self, version: str, wait_for_seconds: int = 0) -> Response:
                 asynchronously, without waiting at all.
         """
         url = f"{API_BASE}/content/{self.subject}/{self.repo}/{self.package}/{version}/publish"
-        res = self.session.post(
-            url, json={"publish_wait_for_secs": wait_for_seconds}
-        )
+        res = self.session.post(url, json={"publish_wait_for_secs": wait_for_seconds})
         res.raise_for_status()
         return res
 
+    def get_metadata(self) -> Response:
+        url = f"{API_BASE}/packages/{self.subject}/{self.repo}/{self.package}"
+        res = self.session.get(url)
```

Comment:
I think your `pyfmt` may have inadvertently introduced this new function. ISTM that it is missing a `raise_for_status()` call?